### PR TITLE
NOX-22 translations get all

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

File: src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Queries/GetWorkplaceTranslationsQuery.g.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+using Microsoft.EntityFrameworkCore;
+
+using Nox.Application.Commands;
+using YamlDotNet.Core.Tokens;
+using ClientApi.Application.Dto;
+using ClientApi.Infrastructure.Persistence;
+
+namespace ClientApi.Application.Queries;
+
+public record  GetWorkplaceTranslationsQuery(System.Int64 keyId) : IRequest <IQueryable<WorkplaceLocalizedDto>>;
```
Comment (rochar) [inline src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Queries/GetWorkplaceTranslationsQuery.g.cs]: To allow extend, although I'm not seeing a useful scenario at the moment
```suggestion
public partial record  GetWorkplaceTranslationsQuery(System.Int64 keyId) : IRequest <IQueryable<WorkplaceLocalizedDto>>;
```

**Review by rochar — DISMISSED** [review]
looks good, just a small adjustment

**Review by rochar — DISMISSED** [review]

File: src/Nox.Generator/Application/Queries/TranslationsQuery.template.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+internal partial class {{className}}Handler:{{className}}HandlerBase
+{
+    public  {{className}}Handler(DtoDbContext dataDbContext): base(dataDbContext)
+    {
+    
+    }
+}
+
+internal abstract class {{className}}HandlerBase:  QueryBase<IQueryable<{{entity.Name}}LocalizedDto>>, IRequestHandler<Get{{entity.Name}}TranslationsQuery, IQueryable<{{entity.Name}}LocalizedDto>>
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Generator/Application/Queries/TranslationsQuery.template.cs]:
```suggestion
internal abstract class {{className}}HandlerBase:  QueryBase<IQueryable<{{entity.Name}}LocalizedDto>>, IRequestHandler<{{className}}, IQueryable<{{entity.Name}}LocalizedDto>>
```

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

File: src/Nox.Generator/Application/Queries/TranslationsQuery.template.cs (hunk 2 above)
Comment (msadikkoseaupa) [reply src/Nox.Generator/Application/Queries/TranslationsQuery.template.cs]: desired change has been applied

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**